pennylane/tape/qscript.py

```diff
@@ -1143,37 +1143,42 @@ def specs(self):
             dict[str, Union[defaultdict,int]]: dictionaries that contain quantum script specifications
 
         **Example**
+        >>> prep = [qml.BasisState(np.array([1,1]), wires=(0,"a"))]
+        >>> ops = [qml.DoubleExcitation(0.1, wires=[0, 1, 2, 3])]
+        >>> qscript = QuantumScript(ops, [qml.expval(qml.PauliZ(0))], prep)
 
-        >>> ops = [qml.Hadamard(0), qml.RX(0.26, 1), qml.CNOT((1,0)),
-        ...         qml.Rot(1.8, -2.7, 0.2, 0), qml.Hadamard(1), qml.CNOT((0,1))]
-        >>> qscript = QuantumScript(ops, [qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))])
-
-        Asking for the specs produces a dictionary as shown below:
+        Asking for the specs produces a dictionary of useful information about the circuit:
 
+        >>> qscript.specs["num_observables"]
+        1
         >>> qscript.specs['gate_sizes']
-        defaultdict(int, {1: 4, 2: 2})
-        >>> qscript.specs['gate_types']
-        defaultdict(int, {'Hadamard': 2, 'RX': 1, 'CNOT': 2, 'Rot': 1})
-
-        As ``defaultdict`` objects, any key not present in the dictionary returns 0.
-
-        >>> qscript.specs['gate_types']['RZ']
-        0
-
+        defaultdict(<class 'int'>, {2: 1, 4: 1})
```

Comment:
Is this supposed to be here?

Reply:
Context: 
While initializing an instance of `Resources` the user can pass any dictionary for the `gate_types` argument. It's  more efficient to use a `defaultdict` in `_count_resources()` when counting the `gate_types`. Due to this, specs will return a `Resources` instance which has specifically a `defaultdict` as the type for the Resources object. 

Answer: 
I updated the doc-string to match the output on terminal